hudi-spark-datasource/hudi-spark/src/main/scala/org/apache/hudi/HoodieSparkSqlWriter.scala

```diff
@@ -377,6 +398,60 @@ object HoodieSparkSqlWriter {
     (syncHiveSuccess, common.util.Option.ofNullable(instantTime))
   }
 
+  def bulkInsertAsRowNoMetaColumns(sqlContext: SQLContext,
```

Comment:
this has lot of commonality between existing bulk_insert for row writer path. Once I get first set of reviews, will unify them.

Reply:
I meant for changes in HoodieSparkSqlwriter. but both uses the same custom datasource.

Reply:
lets please unify this.